internal/lsp/server.go

```diff
@@ -121,6 +134,8 @@ func (l *LanguageServer) Handle(
 		return struct{}{}, nil
 	}
 
+	fmt.Fprintf(l.errorLog, "%s\n", string(util.MustMarshalJSON(req)))
```

Comment:
```suggestion
	fmt.Fprintln(l.errorLog, string(util.MustMarshalJSON(req)))
```
(if it was desired to keep this line)

Reply:
Thanks, I don't think we need this tbh. Will remove, I have some ideas to improve the logging in the server, but will do in another PR.